src/scripts/markeranimation.js

```diff
@@ -1,33 +1,54 @@
-import { Settings } from './settings.js';
+import {Settings} from './settings.js';
 
 export class MarkerAnimation {
     /**
      * Starts the animation loop
      */
-    static startAnimation() {
-        if (!this.animator) {
-            this.animator = this.animateRotation.bind(this);
-            canvas.app.ticker.add(this.animator);
+
+    static startAnimation(marker_type = "turnmarker") {
+        if (!this.animators) {
+            this.animators = {};
         }
-        return this.animator;
+        this.animators[marker_type] = this.animateRotation.bind(this, marker_type);
```

Comment:
Now we pass marker_type to animate rotation, and instead of having one ticker for every animation, we split it up.

This is probably less optimized, but this makes it easier to compartmentalize additional animators for other possible marker types.